recipes/hdf5/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+if [ `uname -m` == ppc64le ]; then
+    B="--build=ppc64le-linux"
```

Comment:
This is a piece of lint from the internal recipe.  We also build some packages for ppc64le.